CONTRIBUTING.md

```diff
@@ -56,13 +56,15 @@ vagrant up
 vagrant ssh
 # you're now in a shell in a virtual machine
 sudo su
+go get github.com/onsi/ginkgo/ginkgo
```

Comment:
To avoid getting every time, could you help to move this two lines to `Vagrantfile`?

Reply:
Done